spec/features/account_reset/delete_account_spec.rb

```diff
@@ -9,6 +9,9 @@
   let(:user_email) { user.email_addresses.first.email }
   let(:push_notification_url) { 'http://localhost/push_notifications' }
 
+  # Force hour so that running this near midnight UTC doesn't fail
+  let(:frozen_now) { Time.zone.now.change(hour: 9) }
```

Comment:
should we specify a particular timezone? Or just pin a specific time? Zonebie will randomize the timezone which can make things trickier

```suggestion
  let(:frozen_now) { ActiveSupport::TimeZone['UTC'].parse('2023-01-01T00:00:00Z') }
```

Reply:
We tried picking a specific date, and had to revert because the test started failing. There's something outside of freeze_time that depends on the date being "now".

Reply:
ok but I think that Zonebie may mess that up, with 24 hours worth of different timezones, I am worried that specifying hour 9 only will not be sufficient

Reply:
Any thoughts on how to fix? Or do we put up with a flaky test if we run at 5pm PDT?